xskillscore/core/utils.py

```diff
@@ -18,7 +21,7 @@ def suppress_warnings(msg=None):
         yield
 
 
-def _preprocess_dims(dim, a):
+def _preprocess_dims(dim: Dim, a: XArray) -> Tuple[List[str], Tuple[int, ...]]:
```

Comment:
I can't remember what `...` (elipsis) means

Reply:
Zachary was kind enough to explain "a tuple of unspecified length containing only things of type int"